src/content/app/genome-browser/contexts/BrowserContext.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import EnsemblGenomeBrowser from '@ensembl/ensembl-genome-browser';
+
+import type { CogList } from '../components/browser-cog/BrowserCogList';
+import type { StateZmenu } from '../components/zmenu/ZmenuController';
+
+type GenomeBrowserContextType = {
+  genomeBrowser: EnsemblGenomeBrowser | null;
+  setGenomeBrowser: (genomeBrowser: EnsemblGenomeBrowser | null) => void;
+  zmenus: StateZmenu;
+  setZmenus: (zmenus: StateZmenu) => void;
+  cogList: CogList | null;
+  setCogList: (cogList: CogList) => void;
+};
```

Comment:
Something went wrong with your conflict resolution. This is what the genome browser context type should look like now:

https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/genome-browser/Browser.tsx#L165-L170

Reply:
I don't see `CogList` in `BrowserContext.tsx`. Are you referring to a different file?

Reply:
✅